docs/content/latest/explore/linear-scalability/sharding-data.md

```diff
@@ -337,56 +294,91 @@ Let's get started:
 
-1. Here, the key has been written to one of the tablets. In this example, the tablet's UUID is `eecd01f0a7cd4537ba571bdb85d0c094`. Check the [table details page](http://127.0.0.1:7000/table?keyspace_name=ybdemo_keyspace&table_name=cassandrakeyvalue) to determine which node this tablet belongs to, and in this case it's `node-1`.
-
-    ![Tablet ownership with auto-sharding](/images/ce/auto-sharding-tablet-ownership.png)
+Here, the key has been written to one of the tablets. In this example, the tablet's UUID is `0df2c7cd87a844c99172ea1ebcd0a3ee`. Check the [table details page](http://127.0.0.1:7000/table?keyspace_name=ybdemo_keyspace&table_name=cassandrakeyvalue) to determine which node this tablet belongs to, and in this case it's `node-1`.
 
-1. You can also easily confirm that `node-1` does have about 10MB of storage being used.
-
-    ![Inserting values with auto-sharding](/images/ce/auto-sharding-single-kv-insert.png)
+![Tablet ownership with auto-sharding](/images/ce/sharding_tablet.png)
 
 ### Automatic sharding when adding nodes
 
 1. Add one more node to the universe for a total of 4 nodes:
 
     ```sh
-    $ ./bin/yb-ctl add_node --tserver_flags "memstore_size_mb=1"
+    $ ./bin/yugabyted start \
+                      --base_dir=/tmp/ybd4 \
+                      --listen=127.0.0.4 \
+                      --join=127.0.0.1 \
+                      --tserver_flags "memstore_size_mb=1"
     ```
 
-1. Check the tablet servers page, to see that the tablets are re-distributed evenly among the 4 nodes and each node now has 3 shards:
+1. Check the tablet servers page, to see that the tablets are re-distributed evenly among the 4 nodes:
 
-    ![Auto-sharding when adding one node](/images/ce/auto-sharding-add-1-node.png)
+    ![Auto-sharding when adding one node](/images/ce/sharding_4nodes.png)
 
 1. Add 2 more nodes to the universe, making it a total of 6 nodes:
 
     ```sh
-    $ ./bin/yb-ctl add_node --tserver_flags "memstore_size_mb=1"
+    $ ./bin/yugabyted start \
+                      --base_dir=/tmp/ybd5 \
+                      --listen=127.0.0.5 \
+                      --join=127.0.0.1 \
+                      --tserver_flags "memstore_size_mb=1"
     ```
 
     ```sh
-    $ ./bin/yb-ctl add_node --tserver_flags "memstore_size_mb=1"
+    $ ./bin/yugabyted start \
+                      --base_dir=/tmp/ybd5 \
+                      --listen=127.0.0.5 \
+                      --join=127.0.0.1 \
+                      --tserver_flags "memstore_size_mb=1"
     ```
 
 1. Verify that the tablets are evenly distributed across the 6 nodes. Each node now has 2 tablets.
 
-    ![Auto-sharding when adding three nodes](/images/ce/auto-sharding-add-3-node.png)
+    ![Auto-sharding when adding three nodes](/images/ce/sharding_6nodes.png)
 
 ### [Optional] Clean up
 
 If you're done experimenting, run the following command to shut down the local cluster:
 
 ```sh
```

Comment:
You could combine all the shell commands into one block...

Reply:
It brings more clarity I believe by saying that we are shutting down nodes individually.